src/pages/players/PlayerList/PlayerList.tsx

```diff
@@ -1,20 +1,23 @@
 import React from "react";
 import { useQuery } from "react-query";
+import { Loading } from "../../../components/Loading";
 import { PageTitle } from "../../../components/PageTitle";
+import { RecentPlayerTable } from "../../../components/RecentPlayerTable";
 import { StatsApi } from "../../../api";
-import { IPlayerDetails } from "../../../api/types";
+import { IRecentPlayer } from "../../../api/types";
+import { LAST_RECENT_PLAYERS_NUM } from "../../../constants";
 
 export const PlayerList: React.FC = () => {
-  // const { isLoading, data } = useQuery<IPlayerDetails[]>(
-  //   ["players-recent-thirty"],
-  //   StatsApi.Players.RecentThirty
-  // );
-
-  // console.log(isLoading, data);
+  const { isLoading, data } = useQuery<IRecentPlayer[]>(
+    ["players-recent-thirty"],
```

Comment:
for later: the query is called players-recent-thirty , but the LAST_RECENT_PLAYERS_NUM is 50